skimage/exposure/_adapthist.py

```diff
@@ -25,7 +25,7 @@
 
 
 @adapt_rgb(hsv_value)
-def equalize_adapthist(image, ntiles_x=8, ntiles_y=8, clip_limit=0.01,
+def equalize_adapthist(image, ntiles_x=None, ntiles_y=None, clip_limit=0.01,
```

Comment:
oh, because people might use keyword arguments without their names? I hadn't thought that changing the order of keyword arguments was consider a break in backwards compatibility...